CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
 
 * **xmlupload:** use custom IRIs created from salsah ARKs for XML upload (DEV-179) ([#147](https://www.github.com/dasch-swiss/dsp-tools/issues/147)) ([873324a](https://www.github.com/dasch-swiss/dsp-tools/commit/873324ad4cd4fefd48d8d2fc37f08774285849d5))
 
-### [1.8.1](https://www.github.com/dasch-swiss/dsp-tools/compare/v1.8.0...v1.8.1) (2022-01-11)
+## [1.8.1](https://www.github.com/dasch-swiss/dsp-tools/compare/v1.8.0...v1.8.1) (2022-01-11)
```

Comment:
the CHANGELOG.md is automatically created. So changing stuff here will lead to inconsistencies in the future, once more automatic stuff is added... I'd suggest not doing this, but it's up to you.  
(In case you're wondering why sometimes the versions are h2 and sometimes they are h3: that's the difference between minor and patch releases.)